sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -20,66 +20,161 @@ Azure Key Vault Certificates Spring Boot Starter is Spring starter for [Azure Ke
 ```
 [//]: # ({x-version-update-end})
 
+### Creating an Azure Key Vault
+
+1. Log into <https://portal.azure.com>.
+
+2. Select **Create a resource**, then eBrowse to the Azure portal atnter `Key Vault` in `Search the Marketplace` and select `Key vault`.
+
+![Find Azure Key Vault Resource 01](resource/find-azure-key-vault-resource-01.png)
+
+![Find Azure Key Vault Resource 02](resource/find-azure-key-vault-resource-02.png)
+
+3. Select **Create**.
+
+![Create new Key Vault](resource/create-new-key-vault.png)
+
+4. On the **Create key vault** page, enter the following information:
+
+   * Choose the **Subscription** you want to use for your namespace.
+   * Specify whether to create a new **Resource group** for your namespace, or choose an existing resource group.
+   * Enter a unique **Key vault name**, which will become part of the URI for your Vault URI.
+   * Specify the **Region** for your Key vault.
+   * Pricing tier.
+   
+   When you have specified the options listed above, select **Review + Create**, review the specifications and select **Create** to Key vault. The deployment will take several minutes.
+
+![Specify the options](resource/specify-the-options.png)
+
+![Create Key Vault resource](resource/create-key-vault-resource.png)
+
+5. When complete, select `Go to resource`.
+
+![Go to resource](resource/go-to-resource.png)
+
+6. When the page for your app registration appears, copy your **Vault URI**;
+
+![Save vault uri](resource/save-vault-uri.png)
+
+7. Click **Certificates** in the left navigation pane.  Then select **Generate/Import**.
+
+![Create Certificates](resource/create-certificates.png)
+
+8. Enter a **Certificates name**, and enter a **Subject** like `CN=mydomain.com`. then select **create**.
+
+![Specify Certificates Info](resource/specify-certificates-info.png)
+
+9. After the certificate is successfully created, it takes a while for the status to become `Enabled`. You can select **refresh** to check current status.
+
+![Check Certificates status](resource/check-certificates-status.png)
+
 ## Key concepts
-This starter is based on a JCA Provider for Azure Key Vault which is a JCA provider for certificates in 
-Azure Key Vault. It is built on four principles:
- 
-1. Must be extremely thin to run within a JVM.
-1. Must not introduce any library version conflicts with Java app code dependencies.
-1. Must not introduce any class loader hierarchy conflicts with Java app code dependencies.
-1. Must be ready for "never trust, always verify and credential-free" Zero Trust environments.
+This starter allows you to securely manage and tightly control your certificates by using Azure Key Vault or side-load certificates by supplying them as part of the application.
 
 ## Examples
 ### Server side SSL
 
-#### Using a managed identity
+#### Using a client ID and client secret
+
+1. Select **Show portal menu**, then **Azure Active Directory**.
```

Comment:
```
1. Click `Azure Active Directory`.
1. Click `App registration`.
1. Click `New registration`
etc..
```

Refs: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-starter-active-directory/README.md#register-the-application-in-azure-ad